app/src/main/java/org/mozilla/focus/browser/binding/AbstractBinding.kt

```diff
@@ -21,6 +22,7 @@ abstract class AbstractBinding(
 ) : LifecycleAwareFeature {
     private var scope: CoroutineScope? = null
 
+    @OptIn(ExperimentalCoroutinesApi::class)
```

Comment:
Alternatively we could also globally optin to the API and wouldn't need to add those annotations everywhere:
https://github.com/mozilla-mobile/android-components/blob/b266ac3d16e15cabc969cfe33c9cf965f557e257/components/feature/prompts/build.gradle#L25-L29